app/src/sharedTest/java/org/oppia/app/topic/reviewcard/ReviewCardFragmentTest.kt

```diff
@@ -0,0 +1,71 @@
+package org.oppia.app.topic.reviewcard
+
+import android.widget.TextView
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withParent
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.Matchers.allOf
+import org.hamcrest.Matchers.instanceOf
+import org.hamcrest.Matchers.not
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.parser.RichTextViewMatcher.Companion.containsRichText
+import org.oppia.app.topic.reviewcard.ReviewCardActivity.Companion.createReviewCardActivityIntent
+import org.oppia.domain.topic.FRACTIONS_TOPIC_ID
+import org.oppia.domain.topic.SUBTOPIC_TOPIC_ID
+
+/** Tests for [ReviewCardFragmentTestActivity]. */
```

Comment:
`/** Tests for [ReviewCardActivity]. */`

Reply:
Done.